driver/esp32/espidf.c

```diff
@@ -203,3 +203,63 @@ void ili9341_flush(void *_disp_drv, const void *_area, void *_color_p)
     ili9341_send_data_dma(disp_drv, color_p, size * 2, dc, *spi_ptr);
 }
 
+
+typedef struct {
+	uint8_t blue;
+	uint8_t green;
+	uint8_t red;
+} lv_color24_t;
```

Comment:
Potential conflict with LVGL if the [`feat/rgb888`](https://github.com/lvgl/lvgl/issues/1722) branch gets merged. I'd suggest just calling this `color24_t` and leave the `lv_` prefix off.

Reply:
OK, no problem...